extensions/wikia/PaidAssetDrop/PaidAssetDropHooks.class.php

```diff
@@ -0,0 +1,39 @@
+<?php
+class PaidAssetDropHooks {
+
+	/**
+	 * Modify assets appended to the bottom of the page
+	 *
+	 * @param array $jsAssets
+	 *
+	 * @return bool
+	 */
+	public static function onOasisSkinAssetGroups( &$jsAssets ) {
+		global $wgPADConfig;
```

Comment:
Uhm, I'm hesitating ;) 

[Before I used `$wgPaidAssetDrop`](https://github.com/Wikia/app/commit/47d479ded93ca21410edb510c3c3874b4cb2d228) but then I thought: "it says nothing about what the variable is for" and I changed it to the one with acronym since `$wgPaidAssetDropConfig` seemed too long for me. We always have documentation and comments in the code telling us what's the acronym stands for. Do you think it's not enough?